docs/projects/earm-pipeline/about.md

```diff
@@ -0,0 +1,41 @@
+---
+id: about
+title: eARM Scientific Workflow with RAG & HPC Deployment
+custom_edit_url: null
+---
+
+## Overview
+
+Adapting and contributing to the open-source eARM molecular modeling workflow by integrating Retrieval-Augmented Generation (RAG) knowledge retrieval capabilities and deploying the scientific workflow on SLU HPC systems.
+
+## Information
+
+- **Source Code:** -[https://github.com/oss-slu/earm-rag-hpc-deployment](https://github.com/oss-slu/earm-rag-hpc-deployment) [<img src="/img/git-alt.svg" alt="git" width="25" height="25" />](https://github.com/oss-slu/earm-rag-hpc-deployment)
```

Comment:
Currently the repo is at https://github.com/oss-slu/earm_pipeline. Please let me know if you would like me to rename it (including different capitalization).